src/explorer.ts

```diff
@@ -190,6 +201,9 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
             this.kind == kuberesources.allKinds.secret ||
             this.kind == kuberesources.allKinds.configMap) {
             treeItem.contextValue = `vsKubernetes.resource.${this.kind.abbreviation}`;
+            if (this.kind === kuberesources.allKinds.pod && this.metadata.status != null) {
```

Comment:
missed this - use the stricter equality check:

`this.metadata.status !== null`

Reply:
Thanks for catching that, fixed this.